README.md

```diff
@@ -1,5 +1,7 @@
 # React-shop-cloudfront
 
+CloudFront URL: djubq1e7163cb.cloudfront.net
+
```

Comment:
Natallia, could you please also add the link to S3 Bucket as it's mentioned in p.1 of [Task 2.3](https://github.com/EPAM-JS-Competency-center/cloud-development-course-initial/blob/main/2_serving_spa/task.md)?